setup.cfg

```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = Twisted
+version = attr: src.twisted.__version__
+description = An asynchronous networking framework written in Python
+author = Twisted Matrix Laboratories
+author_email = twisted-python@twistedmatrix.com
+maintainer = Glyph Lefkowitz
+maintainer_email = glyph@twistedmatrix.com
+url = https://twistedmatrix.com/
+project_urls =
+    Documentation=https://twistedmatrix.com/documents/current/
+    Source=https://github.com/twisted/twisted
+    Issues=https://twistedmatrix.com/trac/report
+license = MIT
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+long_description_content_type = text/x-rst
+
+[options]
+python_requires = >=3.5
```

Comment:
Since `typing.TYPE_CHECKING` was added in 3.5.2, I think the minimum version needs a tiny bump.

Reply:
doesn't this stuff get backported without changing the version in some platforms?

I think it will be changed to 3.6 soon though

Reply:
- With `>=3.5` on a system that runs actual Python 3.5.0, `pip` will install the new Twisted release and then it will break when importing `typing.TYPE_CHECKING`.
- With `>=3.5.2` on a system that runs Python that identifies as 3.5.0 but is actually patched with bits from later releases, `pip` won't install the new Twisted release.

Since the latter is the least disruptive for the user, I think it's preferable.

There are plans to drop 3.5, see #1386, but that isn't scheduled to be merged until after the next release.

Reply:
I've pushed that change now